### DISCUSSION TITLE
8月25日退赛事件全梳理 | osu! FumoFumo Cup

### DISCUSSION THREAD

**Discussion by giscus** [offfc/offfc.github.io#37] [General]
# 8月25日退赛事件全梳理 | osu! FumoFumo Cup

退赛选手与主办方已经就此事件达成一致，双方和平解决，并且后续不再就此事发生争执，本梳理将会尽量做到客观公正，作留档用。

https://offfc.github.io/blog/history/

**Comment by CloneWith** [comment]
支持*和平客观*的处理方式
各位的肯定与支持，会让OFFC以及整个osu!社区变得更好
